3-fetch-api-ajax/fetch.js

```diff
@@ -0,0 +1,27 @@
+
+'use strict'
+
+const URLS = 'https://api.github.com/repos/push-dev/frontend-roadmap/commits';
+
+function loadJson(url) {
+    return fetch(url)
+      .then(response => response.json());
+  }
+
+   async function showAvatar(githubUser) {
+    return new Promise(function(resolve, reject) {
+      let img = document.createElement('img');
+      let paragraph = document.createElement('h1');
+      img.src = githubUser.avatar_url;
+      img.className = "promise-image";
+      paragraph.innerHTML = 'Name: ' + githubUser.login;
+      document.body.append(paragraph);
+      document.body.append(img);     
```

Comment:
Please implement this exercise without creating these elements, they should be already present on the HTML and you just need to use the reference to them.